package.json

```diff
@@ -16,19 +16,16 @@
     "README.md",
     "dist"
   ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/wowserhq/wowser-math.git"
-  },
+  "repository": "wowserhq/wowser-math",
   "keywords": [
     "wowser"
   ],
-  "author": "",
+  "author": "fallenoak",
```

Comment:
I'd suggest we do something like:

```
"author": "Wowser Contributors"
```

In our case, I'd call the group of developers that write the package the author, even if that's not a single person in a physical sense. Something like corporate personhood, I guess. :D

Grunt, for example, does something similar: https://github.com/gruntjs/grunt/blob/master/package.json#L5

Reply:
Nice, should we make sure the LICENSE and this author field are in sync? Either `The Wowser Contributors` or `Wowser Contributors`?

Reply:
I guess I'd vote for `Wowser Contributors` (no the). Want to adjust the line in `LICENSE`?